package.json

```diff
@@ -31,10 +31,13 @@
 		"deserialize"
 	],
 	"dependencies": {
-		"type-fest": "^0.12.0"
+		"buffer": "^5.5.0",
+		"type-fest": "^0.12.0",
+		"util": "^0.12.2"
```

Comment:
`buffer` and `util` are core modules. They should not be here.

Reply:
@sindresorhus , wouldn't they be needed for frontend?

Reply:
No, it's up to browser user to polyfill those. Webpack does it automatically, I think.

Reply:
fix